sample/DateTimeParser/DateTimeParser.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using Superpower;
+using Superpower.Parsers;
+
+namespace DateTimeParser
+{
+    static public class DateTimeParser 
+    {
+        private static TextParser<string> _2DigitParser =
```

Comment:
The codebase doesn't use explicit `private` modifiers, so it might save some space to lop them off, here.